vector/src/main/java/im/vector/util/PreferencesManager.java

```diff
@@ -132,6 +131,10 @@
 
     private static final String SETTINGS_VIBRATE_ON_MENTION_KEY = "SETTINGS_VIBRATE_ON_MENTION_KEY";
 
+    // Analytics keys (Piwik, Matomo, etc.)
+    public static final String SETTINGS_USE_ANALYTICS_KEY = "SETTINGS_USE_ANALYTICS_KEY";
+    public static final String SETTINGS_AUTHORIZATION_TO_USE_ANALYTICS_REQUESTED = "SETTINGS_AUTHORIZATION_TO_USE_ANALYTICS_REQUESTED";
```

Comment:
This string is not a settings, but a Key for SharedPrefs. It has to be moved elsewhere